Sources/FBLPromises/FBLPromise.m

```diff
@@ -298,6 +298,7 @@ - (FBLPromise *)chainOnQueue:(dispatch_queue_t)queue
 extern void FBLIncludeTimeoutCategory(void);
 extern void FBLIncludeValidateCategory(void);
 extern void FBLIncludeWrapCategory(void);
+extern void FBLIncludeTestingCategory(void);
```

Comment:
Alphabetize along with the rest, here and below.

Reply:
Done in 6592eea